source/ui/BioXAS/BioXASControlEditor.cpp

```diff
@@ -365,6 +379,15 @@ void BioXASControlEditor::onCalibrateActionTriggered()
 	}
 }
 
+void BioXASControlEditor::onPropertiesActionTriggered()
+{
+	if (control_) {
+		AMControlView *view = new AMControlView(control_);
+		view->setWindowTitle(QString("%1 properties").arg(title()));
+		view->show();
```

Comment:
You may want to add `view->raise();` to guarantee that it shows up.  

Also, this is a memory leak from what I can tell as nothing keeps track of this pointer.